### ISSUE TITLE
ClearScreenDemo not running on Linux

### ISSUE THREAD

**Issue by nroduit** [LWJGLX/lwjgl3-awt#59]
When running ClearScreenDemo on Linux (Ubuntu 22.04.1, X11) several times, I get different errors:

1. Exception in thread "main" java.lang.AssertionError: Failed to create swap chain: Initialization of an object could not be completed for implementation-specific reasons.

2. Xlib:  extension "GLX" missing on display ":1".
Exception in thread "main" java.lang.AssertionError: Failed to query number of physical device surface formats: Initialization of an object could not be completed for implementation-specific reasons.
	at org.lwjgl.vulkan.awt.ClearScreenDemo.getColorFormatAndSpace(ClearScreenDemo.java:239)
	
3. [xcb] Unknown request in queue while dequeuing
[xcb] Most likely this is a multi-threaded client and XInitThreads has not been called
[xcb] Aborting, sorry about that.
java: ../../src/xcb_io.c :175 : dequeue_pending_request:  l'assertion « !xcb_xlib_unknown_req_in_deq » a échoué.	

My goal was to evaluate the use of Vulkan. I guess the different errors are due to the asynchronous behavior of the Vulkan API, are there any other recommendations?

Here is my graphics drivers 
```
$ nvidia-smi
Sun Sep 11 09:03:34 2022       
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 470.141.03   Driver Version: 470.141.03   CUDA Version: 11.4     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                               |                      |               MIG M. |
|===============================+======================+======================|
|   0  Quadro P2000        Off  | 00000000:18:00.0  On |                  N/A |
| 52%   44C    P0    18W /  75W |    654MiB /  5025MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
```

**Comment by SWinxy** [comment]
Does `SimpleDemo.java` work? If it does, then it means that the problem is probably with the demo.

**Comment by nroduit** [comment]
Yes, `SimpleDemo.java` works without any problem. However, it just creates an instance.

**Comment by SWinxy** [comment]
Since it's been a week without me doing anything, but I'm going to keep this open as a reminder to fix it. Since lwjgl3-awt doesn't change anything other than the surface creation, there isn't a need for a Vulkan demo going further when there are demos at [LWJGL/lwjgl3-demos](https://github.com/LWJGL/lwjgl3-demos/tree/main/src/org/lwjgl/demo/vulkan) and [this excellent guide](https://github.com/lwjglgamedev/vulkanbook).

Thanks for raising the issue though. It's something I can look into when I feel really bored.
